docs/Config_Changes.md

```diff
@@ -135,8 +135,8 @@ packages (e.g. `sudo apt install python3-numpy python3-matplotlib`) to
 continue using these scripts. For more details, refer to
 [Software installation](Measuring_Resonances.md#software-installation).
 Alternatively, users can temporarily force the execution of these scripts
-under Python 2 by explicitly calling Python2 interpretor in the console:
-`python2 ~/klipper/scripts/calibrate_shaper.py ...`
+under Python 2 by explicitly calling Python3 interpretor in the console:
+`python3 ~/klipper/scripts/calibrate_shaper.py ...`
```

Comment:
self-review:

**Problem this submission fixes:**
This python file being executed appears to have been updated to python3 since 2021, https://github.com/Klipper3d/klipper/commit/309b8e030f68329d04573042ffc53a2745ac088b

Most UNIX systems don't ship with python2 any more, (and it's getting harder to maintain the python2 ecosystem), so this line trips up anyone trying to follow the instructions.

This small submission is high impact to the audience of users who are following this documentation.